### ISSUE TITLE
`KeyPair` defaults

### ISSUE THREAD

**Issue by mversic** [hyperledger/iroha#2101]
After #2077 is merged implementation of the `KeyPair::Default` will contain empty vectors for public/private key pair payloads as can be seen in the implementation. I think we need better defaults. Let's agree on a specific valid public/private key pair that we'll use as a default. Also, maybe coordinate with other teams and have this keypair exposed in documentation

**Comment by appetrosyan** [comment]
The de-facto standard is the key pair we use in `genesis` and `config`. 
```
ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0
```

**Comment by appetrosyan** [comment]
Perhaps a similar trait that is not `Default` but acts like one.

**Comment by appetrosyan** [comment]
Addressed in #2188
